index.html

```diff
@@ -230,23 +230,47 @@ <h3 class="teamsubtitle">Lorem ipsum dolor sit amet proin gravida nibh vel velit
             </div>  
             
             <div class="mainworks">
-                    <div class="itmnw1">
-                        <img class="imgworks" src="../Tajam/assets/images/Works/1.png" alt="1">
-                        <img class="imgworks" src="../Tajam/assets/images/Works/2.png" alt="2">
-                        <img class="imgworks" src="../Tajam/assets/images/Works/3.png" alt="3">
-                        <img class="imgworks" src="../Tajam/assets/images/Works/4.png" alt="4">
-                        <img class="imgworks" src="../Tajam/assets/images/Works/5.png" alt="5">
-                        <img class="imgworks" src="../Tajam/assets/images/Works/6.png" alt="6">
-                    </div>
+                <div class="mainworks-top">
```

Comment:
better will be using one flex container for all images

Reply:
When one container is used, the distance between the photos is available. margin or padding does not help. And when the flex-wrap function is turned on: wrap photos become dynamic and change their position, and I need them to be static.